daft/daft.pyi

```diff
@@ -34,4 +34,19 @@ class ImageMode(Enum):
         """
         ...
 
+class ImageFormat(Enum):
```

Comment:
We shouldn't need to have the image format right? right now we're pulling data of mixed extensions from s3.